LadderManager.cpp

```diff
@@ -681,13 +716,12 @@ ResultType LadderManager::StartGame(BotConfig Agent1, BotConfig Agent2, std::str
 			std::cout << "Create game successful" << std::endl << std::endl;
 		}
 	}
-	std::cout << "Starting bot: " << Agent1.BotName << " with command:"<<std::endl;
-	std::cout << Agent1Path << std::endl;
-	auto bot1ProgramThread = std::async(&StartBotProcess, Agent1Path);
-	std::cout << "Starting bot: " << Agent2.BotName << " with command:" << std::endl;
-	std::cout << Agent2Path << std::endl;
-	auto bot2ProgramThread = std::async(&StartBotProcess, Agent2Path);
-	sc2::SleepFor(1000);
+	auto bot1ProgramThread = std::async(&StartBotProcess, Agent1, Agent1Path);
+	auto bot2ProgramThread = std::async(&StartBotProcess, Agent2, Agent2Path);
+	sc2::SleepFor(500);
+	sc2::SleepFor(500);
```

Comment:
Is two sleeps in a row any different from one sleep for double the time?

Reply:
Yes. Often with 1000 (even 2000) only one of the async threads got called. With two 500 it was always both. Not a problem for the manager itself but this messes up the stdout messages.

Reply:
Interesting. Do we know why this is? Or is it "just the way it works"?